services/mqtt/service.go

```diff
@@ -0,0 +1,151 @@
+package mqtt
+
+import (
+	"fmt"
+	"log"
+	"sync/atomic"
+	"time"
+
+	pahomqtt "github.com/eclipse/paho.mqtt.golang"
+	"github.com/influxdata/kapacitor/alert"
+)
+
+type QOSLevel byte
+
+const (
+	// best effort delivery. "fire and forget"
+	ATMOSTONCE QOSLevel = iota
```

Comment:
No need for constants to be all upper case in Go, go ahead and use Upper Camel Case here.